.github/workflows/continuous-integration-e2e.yaml

```
@@ -65,8 +65,8 @@ jobs:
         CHAIN_HISTORY_PROVIDER: 'http'
         CHAIN_HISTORY_PROVIDER_PARAMS: '{"baseUrl":"http://localhost:4000/chain-history"}'
         DB_SYNC_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/cexplorer'
-        PROJECTION_PG_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/projection'
-        PROJECTION_TEST_PG_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/projection_test'
+        STAKE_POOLS_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/pools'
```

```suggestion
        STAKE_POOL_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/stake_pool'
```
The convention here should be the full name of the provider (_stake pool_)

---

cba27e3cf should solve this